packages/holocron/src/holocronModule.jsx

```diff
@@ -55,9 +56,14 @@ export default function holocronModule({
         }
       }
 
-      componentWillReceiveProps(nextProps) {
-        if (shouldModuleReload && shouldModuleReload(this.props, nextProps)) {
-          const newLoadCount = this.state.loadCount + 1;
+      // ignoring to support deprecated componentWillReceiveProps.
+      // This needs to be removed to support React17
+      // eslint-disable-next-line camelcase
+      UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
could this become `componentDidUpdate` instead ?

Reply:
Updating to `componentDidUpdate ` et al. should be a new effort to refactor Holocron to the proper lifecycle hooks when there's time to understand the implications of those changes.